app/src/main/java/com/github/steroidteam/todolist/model/TodoRepository.java

```diff
@@ -77,4 +83,27 @@ public void setTaskDueDate(int index, Date dueDate) {
                 .thenAccept(this.oneTodoList::setValue);
         this.database.getTodoList(todoListID).thenAccept(this.oneTodoList::setValue);
     }
+
+    private void setTagsList() {
+        List<UUID> tagsIds = oneTodoList.getValue().getTagsIds();
+        System.out.println("Tags IDs : " + tagsIds);
```

Comment:
Why printlns here ?

Reply:
That was for fixing a bug, I'll remove it when the PR's ready for reviews